po/fr.po

```diff
@@ -659,26 +746,32 @@ msgstr "Minimiser la borne inférieure"
 msgid "Maximize Upper Bound"
 msgstr "Maximiser la borne supérieure"
 
-#: src/bz-transaction-manager.c:1080
+#: src/bz-transaction-manager.c:1090
 #, c-format
 msgid "Finished in %.02f seconds"
 msgstr "Terminé en %.02f secondes"
 
-#: src/bz-transaction-view.blp:143
-msgid "Finished Tasks"
-msgstr "Tâches terminées"
+#: src/bz-transaction-view.blp:87
+msgid "Install Size"
+msgstr "Taille après installation"
 
-#: src/bz-transaction-view.c:111
-#, c-format
-msgid "%s to download"
-msgstr "%s à télécharger"
+#: src/bz-transaction-view.blp:173
+msgid "Updated Size"
+msgstr "Taille après mise à jour"
 
-#: src/bz-transaction-view.c:121
-#, c-format
-msgid "%s to install"
-msgstr "%s à installer"
+#: src/bz-transaction-view.blp:186 src/bz-window.blp:299
+msgid "Update"
```

Comment:
There should be two different strings for "Update" as it is used both as a noun and as an infinitive.
The noun is translated into French as "Mise à jour" and the infinitive as "Mettre à jour".

I decided to fall back to the infinitive as it would otherwise be a bit awkward.